### DISCUSSION TITLE
Setting language in init.lua

### DISCUSSION THREAD

**Discussion by uyha** [neovim/neovim#14007] [Q&A] [answered]
I'm trying to migrate from `init.vim` to `init.lua` and I'm stuck at setting the language (locale). In `init.vim`, I can do `lan en_US.utf8` but I couldn't find a way to do the same thing in `init.lua`. Could someone give me a pointer on how to do this?

**Comment by fsouza** [comment, marked as answer]
You can use the function `vim.cmd` to run commands, so for language you could do:

```lua
vim.cmd('lan en_US.utf-8')
```

(on single arg function-calls, parenthesis are optional, so you could also do `vim.cmd 'lan en_US.utf-8'` or `vim.cmd'lan en_US.utf-8'` if that's a style you prefer)

**Reply by uyha** [reply]
Thank you for the answer. I have one further question. This is basically calling vim script in Lua, is there a way to to this using purely Lua? Or currently, the Lua API in Neovim doesn't support it?

**Reply by fsouza** [reply]
Yeah, that command isn't currently available in the API as far as I know.
